WebRoot/css/zm.css

```diff
@@ -1395,6 +1403,18 @@ INPUT[type="text"].search_input-expanded:focus {
 	@AddreddBubbleLeftChild@
 }
 
+.attachmentBubble > :first-child {
+	@AttachmentLeftChild@
+}
+
+.attachmentBubble .AttLink {
+	@AttachmentLinkFontSize@
+}
+
+span.attachmentBubble  span:last-child {
+	@AttachmentFileSize@
```

Comment:
are we sure last span will always be showing file size? any skin can override template to change location of file size span